packages/manager/apps/web/client/app/email-domain/upgrade/upgrade.html

```diff
@@ -0,0 +1,78 @@
+<div data-ovh-alert="{{ $ctrl.alerts.main }}"></div>
+
+<div class="container-fluid">
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="email_domain_upgrade_title"></span>
+    </oui-back-button>
+
+    <oui-stepper>
+
+        <oui-step-form data-header="{{ ::'email_domain_upgrade_step_header_offer' | translate }}"
+            data-editable="!$ctrl.loading.upgrading" data-valid="$ctrl.model.offer"
+            data-prevent-next="!$ctrl.model.offer"
+            data-on-focus="$ctrl.displaySelectedOffer = false"
+            data-on-submit="$ctrl.displaySelectedOffer = true">
+            <p data-translate="email_domain_upgrade_step_header_offer_description"></p>
+            <div class="row">
+                <oui-select-picker
+                    data-ng-if="$ctrl.displaySelectedOffer"
+                    class="mb-3 col-md-4"
+                    data-label="{{ $ctrl.model.offer.displayName }}"
+                    name="{{:: $ctrl.model.offer.name }}"
+                    data-model="$ctrl.model.offer"
+                    data-values="[$ctrl.model.offer]"
```

Comment:
is there only one offer to select? why using a select then?

Reply:
This is the display when the `step-form` is validated to only display the selected offer, it is just for display purpose, when this occurs the user is not selecting the offer.
This is to keep harmonization with the way it is displayed when the steps is being edited. 
Moreover it's the way it's done in public-cloud so I think it's better to have the same experience everywhere